addons/delivery/views/delivery_view.xml

```diff
@@ -22,72 +22,59 @@
-                                    <field name="active"/>
-                                    <field name="sequence"/>
+                                <group attrs="{'invisible':[('delivery_type', '!=', 'fixed')]}">
+                                    <group>
+                                        <field name="fixed_price"/>
+                                    </group>
+                                    <group>
+                                        <div class="o_row">
+                                            <label for="free_if_more_than"/>
+                                            <field name="free_if_more_than"/>
+                                            <field nolabel="1" name="amount" attrs="{'required':[('free_if_more_than','&lt;&gt;',False)], 'invisible':[('free_if_more_than','=',False)]}"/>
+                                        </div>
+                                    </group>
+                                </group>
+                                <group name="general" attrs="{'invisible':[('delivery_type', '!=', 'base_on_rule')]}">
+                                    <field name="price_rule_ids" nolabel="1"/>
                                 </group>
-                             </group>
-                             <notebook>
-                                <page string="Grid definition">
-                                   <field name="line_ids"/>
-                                </page>
-                                <page string="Destination">
+                            </page>
+                            <page string="Destination">
+                                <group>
                                     <group>
-                                        <group>
-                                            <field name="country_ids" widget="many2many_tags"/>
-                                            <field name="state_ids" widget="many2many_tags"/>
-                                        </group>
-                                        <group>
-                                            <label for="zip_from" string="Zip"/>
-                                            <div>
-                                                <field name="zip_from" class="oe_inline"/>
-                                                -
-                                                <field name="zip_to" class="oe_inline"/>
-                                            </div>
-                                        </group>
+                                        <field name="country_ids" widget="many2many_tags"/>
+                                        <field name="state_ids" widget="many2many_tags"/>
                                     </group>
-                                </page>
-                            </notebook>
-                        </form>
-                   </field>
-                   </sheet>
+                                    <group></group>
```

Comment:
Hello @mge-odoo ,
Need to set the Country,state and zip view,
without Empty group : https://drive.google.com/file/d/0B4ek8t9nTzXiaHVkNklXSWFjYU0/view
With Empty group: https://drive.google.com/file/d/0B4ek8t9nTzXiQ0xxTmZvaUY3LU0/view
Thank